src/app/settings/base-stix.component.ts

```diff
@@ -249,4 +250,76 @@ export class BaseStixComponent {
         );
     }
 
+    
+    /**
+     * @param  {any} model (Any legacy STIX model)
+     * @returns boolean
+     * @description This is used to disable the save button in the CRUD pages
+     */
+    public invalidate(model: any): boolean {
+        this.validationErrorMessages = [];
+        let invalid = false;
+
+        // created_by_ref not on identity
+        if (model.type !== 'identity' && (!model.attributes.created_by_ref || !model.attributes.created_by_ref.length)) {
+            this.validationErrorMessages.push('Submitter Organization is required');
+            invalid = true;
+        }
+        
```

Comment:
Probably doesn't matter for the sake of the UI, but I could use dev tools to edit these values on-screen to something invalid, or do it before the AJAX request is submitted. A created-by-ref with one whitespace character defeats this check (same is true for the other values below).

Again, this is probably a moot point for the UI, since a) for all I know, the API defeats these attempts, and b) it's an awful lot of work for no gain, so why do it? Not a stickler for these kinds of things.